docs/source/guides/contributor/chapters/plugins.rst

```diff
@@ -183,6 +183,35 @@ has to be executed early in the code so try to keep the required deps
 minimal (for example the `avocado.core.settings.settings` is not yet
 available).
 
+Plugins execution order
+=======================
+Avocado execute plugins serially one after each other. Because of that, 
+there are cases where the order of executions is important. For example, 
+package plugin installs necessary dependencies and sysinfo plugin collects 
+system information to check if the dependencies were installed. To ensure that 
+package plugin will be executed before sysinfo plugin, avocado is introducing 
```

Comment:
Because this documentation will outlive the introduction of the default priority feature, it would be better IMO to simply state that Avocado allows for plugins to define their default priority (dropping the "is introducing" part).

Also, the order of execution can still be configured by a setting, and how the default priority and order work together need to be made clearer here IMO.

Reply:
Yes, I will update that.